.travis.yml

```diff
@@ -8,12 +8,16 @@ addons:
 env:
   global:
     - DISPLAY=:99.0
-  matrix:
-    - TOXENV=flake8
 matrix:
   include:
     - python: 2.7
+      env: TOXENV=flake8
+    - python: 2.7
       env: TOXENV=py27
+    - python: 3.5
+      env: TOXENV=flake8
+    - python: 3.5
+      env: TOXENV=py35
```

Comment:
As above, this shouldn't be needed. See https://github.com/pytest-dev/pytest-html/blob/master/.travis.yml for an example of using multiple python versions.